l10n_it_fatturapa_in/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     'name': 'Italian Localization - Fattura elettronica - Ricezione',
-    'version': '10.0.1.4.1',
+    'version': '10.0.1.4.2',
```

Comment:
In my opinion, it is not needed to increment version when adding a test.

Reply:
I share this view.

Reply:
So you want to produce a `dev` release?

Reply:
yes @eLBati, in my opinion adding/editing a test does not deserve a new version: new versions are for new behaviors or bug fixes.

Reply:
@taifu @Byloth @labaggio what do you think?

Reply:
In this case, I can agree..  only this case ;)

Reply:
😂 😂